# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: browser/base/content/test/performance/browser_startup_syncIPC.js
# Commit: ddce7f4b64d9
# Full Hash: ddce7f4b64d96e68a86d7a519f1828d06e37176d
# Author: Chris Martin <cmartin@mozilla.com>
# Date: 2020-02-13 18:21:33
# Regressor Bug: 1604412
# File Overlap Count: 2
# Description:
#   Bug 1604412 - Enable remote backbuffer GDI compositing r=jmathies,jld
#   
#   This change adds new "remote backbuffer" logic when compositing without
#   HW acceleration on Windows (IE compositing through Cairo using the Win32
#   GDI)
# ==============================================================================

diff -r ecc5c853bb77 -r ddce7f4b64d9 browser/base/content/test/performance/browser_startup_syncIPC.js
--- a/browser/base/content/test/performance/browser_startup_syncIPC.js	Thu Feb 13 14:26:51 2020 +0000
+++ b/browser/base/content/test/performance/browser_startup_syncIPC.js	Thu Feb 13 14:32:37 2020 +0000
@@ -82,6 +82,12 @@
       maxCount: 3,
     },
     {
+      name: "PCompositorWidget::Msg_Initialize",
+      condition: WIN,
+      ignoreIfUnused: true, // Only on Win10 64
+      maxCount: 3,
+    },
+    {
       name: "PGPU::Msg_AddLayerTreeIdMapping",
       condition: WIN,
       ignoreIfUnused: true, // Only on Win10 64
@@ -143,6 +149,12 @@
       maxCount: 1,
     },
     {
+      name: "PCompositorWidget::Msg_Initialize",
+      condition: WIN,
+      ignoreIfUnused: true, // Only on Win10 64
+      maxCount: 1,
+    },
+    {
       name: "PCompositorBridge::Msg_WillClose",
       condition: WIN,
       ignoreIfUnused: true, // Only on Win7 32
@@ -217,6 +229,12 @@
       maxCount: 1,
     },
     {
+      name: "PCompositorWidget::Msg_Initialize",
+      condition: WIN && WEBRENDER,
+      ignoreIfUnused: true, // Intermittently occurs in "before handling user events"
+      maxCount: 1,
+    },
+    {
       name: "PCompositorBridge::Msg_MapAndNotifyChildCreated",
       condition: WIN,
       ignoreIfUnused: true,